packages/css/utils/breakpoint.scss

```diff
@@ -4,7 +4,7 @@
  */
 
 /*
-  The breakpoint is 820px / 16 = 51.25rems
+  The breakpoint is 854px / 16 = 53.375rems
   https://developer.mozilla.org/en-US/docs/Web/CSS/@media#accessibility_concerns
 */
-$amsterdam-breakpoint: 51.25rem;
+$amsterdam-breakpoint: 53.375rem;
```

Comment:
Looks like an arbitrary value. maybe a descriptive comment?

Reply:
Hm, I'm not sure what to add here aside from the comments at the start of the file and the breakpoint docs in [this PR](https://github.com/Amsterdam/design-system/pull/342). Any suggestions?